lib/main.dart

```diff
@@ -403,38 +403,42 @@ class _MyHomePageState extends State<MyHomePage> with WidgetsBindingObserver {
             })),
         drawer: makeDrawer(context),
         body: BiforgePage(
-          scrollChild: true,
-          showDebugInfo: true,
-          child: Center(
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [
-                // exchange widget
-                _userInfo != null ? ExchangeWidget(_websocket) : SizedBox(),
-                // home screen buttons
-                Visibility(
-                    visible: _userInfo != null,
-                    child: LayoutBuilder(builder: (context, constraints) {
-                      if (constraints.maxWidth < cfg.MaxColumnWidth)
-                        return Column(
-                            mainAxisSize: MainAxisSize.min,
-                            children: [
-                              buttonRow1,
-                              VerticalSpacer(),
-                              buttonRow2
-                            ]);
-                      else
-                        return Row(
-                            mainAxisAlignment: MainAxisAlignment.center,
-                            children: [
-                              buttonRow1,
-                              SizedBox(width: 50),
-                              buttonRow2
-                            ]);
-                    }))
-              ],
-            ),
-          ),
-        ));
+            scrollChild: true,
+            showDebugInfo: true,
+            child: Center(
+              child: LayoutBuilder(builder: (context, constraints) {
+                return Column(children: [
+                  VerticalSpacer(
```

Comment:
@4855062503 if the balance widget goes above this spacer it will look good i think